backend/models/user.go

```diff
@@ -18,6 +18,8 @@ type User struct {
 	Mnemonic          string    `json:"-" gorm:"column:mnemonic"`
 	SSHKey            string    `json:"ssh_key"`
 	Debt              float64   `json:"debt"`
+	Sponsored         bool      `json:"sponsored"`
+	Address           string    `json:"address" gorm:"column:address"`
```

Comment:
What is the address for?

Reply:
or is that the publicAddress of the user keypair? or is that the accountAddress?